### PR TITLE
Spurious dragon changes

### PR DESCRIPTION
**Pull request by pirapira**
I tried to capture the "Spurious Dragon" changes.

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -41,7 +41,7 @@
 \newcommand*\ie{i.e.\@\xspace}
 %\renewcommand{\itemhook}{\setlength{\topsep}{0pt}  \setlength{\itemsep}{0pt}\setlength{\leftmargin}{15pt}}
 
-\title{Ethereum: A Secure Decentralised Generalised Transaction Ledger \\ {\smaller \textbf{EIP-150 revision (\YellowPaperVersionNumber{})}}}
+\title{Ethereum: A Secure Decentralised Generalised Transaction Ledger \\ {\smaller \textbf{EIP-170 revision (\YellowPaperVersionNumber{})}}}
```
Comment (gavofyork) [inline Paper.tex]: 155/160/161 changes, no?

File: Paper.tex [hunk 2]
```diff
@@ -233,6 +233,30 @@ \subsection{World State} \label{ch:state}
...
+\wedge \boldsymbol{\sigma}[a]_b = 0
+\end{equation}
+
+An account is dead when its account state is non-existent or empty:
+\begin{equation}
+\mathtt{\tiny DEAD}(\boldsymbol{\sigma}, a)
+\quad\equiv\quad \boldsymbol{\sigma}[a] = \varnothing \vee \mathtt{\tiny EMPTY}(\boldsymbol{\sigma}, a)
+\end{equation}
+
+Here is a function that updates the balance of an account state but deletes the account state when it becomes empty:
```
Comment (gavofyork) [inline Paper.tex]: "Here is a" -> "We define the"

**Comment by pirapira** [comment]
I fixed these points.

File: Paper.tex [hunk 3]
```diff
@@ -233,6 +233,30 @@ \subsection{World State} \label{ch:state}
...
+
+We define the function that updates the balance of an account state but deletes the account state when it becomes empty:
+\begin{equation}
+\rho_b(\boldsymbol{\sigma}, a, x) \equiv
+\begin{cases}
+\varnothing \quad\text{if}\quad \boldsymbol{\sigma}[a]_n = 0 \wedge x = 0 \wedge \boldsymbol{\sigma}[a]_c = \texttt{\small KEC}\big(()\big) \\
+(\boldsymbol{\sigma}[a]_n, x, \boldsymbol{\sigma}[a]_s, \boldsymbol{\sigma}[a]_c) \quad \text{otherwise}
+\end{cases}
+\end{equation}
+for $a \in \mathbb{B}_{20}$ and $x \in \mathbb{P}_{256}$.
```
Comment (gavofyork) [inline Paper.tex]: "for" -> "where"

File: Paper.tex [hunk 4]
```diff
@@ -704,9 +729,12 @@ \section{Contract Creation} \label{ch:create}
 Code execution depletes gas, and gas may not go below zero, thus execution may exit before the code has come to a natural halting state. In this (and several other) exceptional cases we say an out-of-gas (OOG) exception has occurred: The evaluated state is defined as being the empty set, $\varnothing$, and the entire create operation should have no effect on the state, effectively leaving it as it was immediately prior to attempting the creation.
 
 If the initialization code completes successfully, a final contract-creation cost is paid, the code-deposit cost, $c$, proportional to the size of the created contract's code:
-\begin{equation}
-c \equiv G_{codedeposit} \times |\mathbf{o}|
-\end{equation}
+\begin{align}
+c \equiv \begin{cases}
+  2^{256} - 1 & \text{if} \quad |\mathbf{o}| \le 24000 \\
```
Comment (gavofyork) [inline Paper.tex]: should be 24576

**Comment by pirapira** [comment]
@gavofyork thanks.  I fixed these two points.

**Pull request merged**